src/relations/postgresql.py

```diff
@@ -96,7 +138,7 @@ def _update_db_connections(self, rel_name, db_conn):
             db_conn: Database connection dict.
         """
         if self.charm._state.database_connections is None:
-            self.charm._state.database_connections = {}
+            self.charm._state.database_connections = {"db": None, "visibility": None}
```

Comment:
suggestion: Right hand side is reused a lot. You can put it into a constant and use it like `self.charm... = dict(DEFAULT_FOO)` so that it will be easier to update in the future,